.github/workflows/test.yml

```diff
@@ -24,22 +24,41 @@ env:
   PROFILE: debug
 
 jobs:
-  checks:
+  clippy:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - name: Run tests
-        run: cargo test --workspace
-      - name: Check code style
-        run: cargo fmt --check --all
-      - name: WASM compile check
-        run: |
-          cd lib/web-playground
-          rustup target add wasm32-unknown-unknown
-          cargo check --target wasm32-unknown-unknown
-      - name: Run VRL tests
-        run: |
-          cd lib/tests
-          cargo run
-      - name: Clippy
+      - name: "clippy"
         run: cargo clippy --workspace --all-targets -- -D warnings
+
+  fmt:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: "fmt"
+        run: cargo fmt --check --all
+
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: "test"
+        run: cargo test --workspace
+
+  vrl:
```

Comment:
Don't love this name... but don't know if I have better suggestion.

Reply:
seems fine to me